src/authorizations/components/redesigned/EditTokenOverlay.tsx

```diff
@@ -9,49 +10,122 @@ import {
   FlexBox,
   AlignItems,
   FlexDirection,
+  InputLabel,
+  SlideToggle,
   ComponentSize,
+  JustifyContent,
+  Button,
+  ComponentColor,
+  Page,
 } from '@influxdata/clockface'
 
 // Types
 import {Authorization} from 'src/types'
 
-interface Props {
+// Actions
+import {updateAuthorization} from 'src/authorizations/actions/thunks'
+
+interface OwnProps {
   auth: Authorization
   onDismissOverlay: () => void
 }
 
-export const EditTokenOverlay: FC<Props> = props => {
+type ReduxProps = ConnectedProps<typeof connector>
+
+type Props = ReduxProps & OwnProps
+
+const labels = {
+  active: 'Active',
+  inactive: 'Inactive',
+}
+
+const EditTokenOverlay: FC<Props> = props => {
   const [description, setDescription] = useState(props.auth.description)
+  const handleInputChange = event => setDescription(event.target.value)
+
+  const handleDismiss = () => props.onDismissOverlay()
 
-  const handleDismiss = () => {
-    props.onDismissOverlay()
+  const changeToggle = () => {
+    const {onUpdate, auth} = props
+
+    onUpdate({
+      ...auth,
+      status: auth.status === 'active' ? 'inactive' : 'active',
+    })
   }
 
-  const handleInputChange = event => {
-    setDescription(event.target.value)
+  const onSave = () => {
+    const {onUpdate, auth} = props
+
+    onUpdate({
+      ...auth,
+      description: description,
+    })
+    handleDismiss()
   }
 
   return (
-    <Overlay.Container maxWidth={830}>
+    <Overlay.Container maxWidth={630}>
```

Comment:
i'm noticing a few changes to overlay widths in PRs lately. is there a general pattern here we can extract into a variable or constant?

Reply:
that's a good point. I am choosing maxWidth number based on the figma design. whichever number makes the ui look closest to this image....https://www.figma.com/proto/1VaHno7NzQTFvzRtCnROay/Token%2FSecret?node-id=703%3A8271&scaling=min-zoom&page-id=423%3A3704&starting-point-node-id=423%3A4342